### ISSUE TITLE
Player count on the grid

### ISSUE THREAD

**Issue by xfer44** [Geekid812/TrackmaniaBingo#68]
Wouldn't it be interesting to know the number of players on each map?

It would be a number on each square of the grid.

It would be nice to see the numbers change when there's a bingo coming up.

**Comment by Geekid812** [comment]
I agree that it would be a good addition and possibly new possibilities for strategies.

Currently however the Bingo plugin does not share with the server which map you are currently playing (it only posts the run you got when completed), so we would need a live status system as a requirement before this.
